airbyte-webapp/src/views/Connector/ServiceForm/FormRoot.tsx

```diff
@@ -57,15 +42,15 @@ const FormRoot: React.FC<FormRootProps> = ({
   const { resetServiceForm, isLoadingSchema, selectedService, isEditMode, formType } = useServiceForm();
 
   return (
-    <FormContainer>
+    <Form className={classnames(styles.formContainer, { [styles.emptyContainer]: !selectedConnector })}>
```

Comment:
ℹ️ Here was some issue with conditionally rendering the form when:
- no formFields provided(connector is not selected): don't show the form since it's empty
- formFields provided(connector is selected): render the form with data
- data is loading: show spinner with message

There was a few options how to deal with that cases:
1. Render conditionally the whole form on upper layer, but then we lose the ability to show the spinner when schema is loading
2. Move spinner out of the form - doesn't sound good, leaving the spinner here we show that form has its own loading state
3. Set `display: none` style to form when no value was provided: we prepare the form, it exist in DOM, and reacts on changes immediately.

I've chosen the 3rd option.

Reply:
Since we decided not to split ConnectorCard on multiple Cards - removed this solution, it's not relevant atm